docker/main/docker-deploy.yml

```diff
@@ -159,26 +159,40 @@ services:
       - myredis_pass
     environment:
       - SERVICE_PACKAGE_NAME=${PYTHON_PACKAGE_NAME_DATA_SERVICE:-dmod.dataservice}
-      - LISTEN_PORT=${DOCKER_DATASERVICE_CONTAINER_PORT:-3015}
-      - SERVICE_SSL_DIR=/ssl/data-service
-      - USER_SECRET_NAME=object_store_exec_user_name
-      - PASSWD_SECRET_NAME=object_store_exec_user_passwd
+      - PORT=${DOCKER_DATASERVICE_CONTAINER_PORT:-3015}
+      - CERT_PATH=/ssl/data-service/certificate.pem
+      - KEY_PATH=/ssl/data-service/privkey.pem
+      - SSL_DIR=/ssl/data-service
```

Comment:
Great question! I was thinking the same thing when I was refactoring this. I was not sure (and didn't lookup) if you can provide root and intermediate x509 certs in separate files along side the private and public key? Meaning, I didn't know if we dropped support for the `SSL_DIR` if this would required bundling the root and intermediate certs with the public key cert and if there were reasons we wouldnt want to do that.

If this isn't an issue, I would love get rid of it!